test/controllers/subscriptionData/CorrespondenceAddressControllerSpec.scala

```diff
@@ -414,7 +416,7 @@ class CorrespondenceAddressControllerSpec extends PlaySpec with GuiceOneServerPe
                   contentAsString(result) must include("You must enter Country")
                 }
                }
-              "throw a validation error, if save fails" in new Setup {
+              "throw a global error, if save fails" in new Setup {
```

Comment:
What is the difference between this test and the one that we have already tested above at line 236?